src/gui/accountsettings.cpp

```diff
@@ -832,8 +832,7 @@ void AccountSettings::slotAccountStateChanged()
         }
         case AccountState::NetworkError:
             showConnectionLabel(tr("No connection to %1.")
-                                    .arg(server),
-                _accountState->connectionErrors());
+                                    .arg(server));
```

Comment:
How about a comment that explains that we explicitly don't want to display an error in this case?

Reply:
Errors are logged independent of the ui